cli/benches/vote.rs

```diff
@@ -0,0 +1,79 @@
+use {
+    criterion::{criterion_group, criterion_main, Criterion},
+    solana_cli::cli::{
+        process_command, request_and_confirm_airdrop, CliCommand, CliConfig,
+    },
+    solana_commitment_config::CommitmentConfig,
+    solana_faucet::faucet::run_local_faucet,
+    solana_rpc_client::rpc_client::RpcClient,
+    solana_rpc_client_nonce_utils::blockhash_query,
+    solana_rpc_client_nonce_utils::blockhash_query::BlockhashQuery,
+    solana_sdk::signature::{Keypair, Signer},
+    solana_streamer::socket::SocketAddrSpace,
+    solana_test_validator::TestValidator,
+};
+
+struct TestSetup {
+    compute_unit_price: Option<u64>,
+}
+
+impl TestSetup {
+    fn new(p: u64) -> TestSetup {
+        Self {
+            compute_unit_price: Some(p),
+        }
+    }
+
+    pub fn run(&self) {
+        let mint_keypair = Keypair::new();
+        let mint_pubkey = mint_keypair.pubkey();
+        let faucet_addr = run_local_faucet(mint_keypair, None);
+        let test_validator = TestValidator::with_no_fees(
+            mint_pubkey,
+            Some(faucet_addr),
+            SocketAddrSpace::Unspecified,
+        );
+
+        let rpc_client =
+            RpcClient::new_with_commitment(test_validator.rpc_url(), CommitmentConfig::processed());
+        let default_signer = Keypair::new();
+
+        let mut config = CliConfig::recent_for_tests();
+        config.json_rpc_url = test_validator.rpc_url();
+        config.signers = vec![&default_signer];
+
+        request_and_confirm_airdrop(&rpc_client, &config, &config.signers[0].pubkey(), 100_000)
+            .unwrap();
+
+        // Create vote account
+        let vote_account_keypair = Keypair::new();
+        config.signers = vec![&default_signer, &vote_account_keypair];
+        config.command = CliCommand::CreateVoteAccount {
```

Comment:
i know it is still WIP, just want to clarify that the aim is to bench [vote instructions](https://github.com/anza-xyz/agave/blob/master/sdk/program/src/vote/instruction.rs#L24), calling CliCommand involves too many overheads that'd distort bench results for ixs.